ui/src/infra-model/list/infra-model-list-view.tsx

```diff
@@ -22,7 +22,7 @@ export type InfraModelListViewProps = Pick<
 export const InfraModelListView: React.FC<InfraModelListViewProps> = (
     props: InfraModelListViewProps,
 ) => {
-    const trackNumbers = useTrackNumbers('DRAFT');
+    const trackNumbers = useTrackNumbers('OFFICIAL');
```

Comment:
Tässä oli ihan suora logiikkamuutos/korjaus bugihkolta vaikuttavaan juttuun. Aiemmin InfraModel-näkymässä haettiin aina luonnostilaiset ratanumerot, mutta tämä vaikutti jo lähtökohtaisesti väärältä koska kyseessä ei ole karttanäkymä (plus monilla käyttäjäryhmillä on oikat inframalleihin mutta ei drafteihin)

Reply:
Tällainen logiikkamuutos/bugikorjaus on toki aikas yllättävä tällaisen pullarin yhteydessä.

Tutkailin hieman: Kun Inframallia tuo, niin tuonnin yhteydessä voi myös luoda tuotavalle inframallille uuden ratanumeron (tai useamman), jotka päätyvät DRAFT-maailmaan, jolloin tässä listausnäkymässä tuskin enää näytettäisiin luotua ratanumeroa oikein. 

Tosin tuohon asti ei tässä haarassa päässyt: "Tuo suunnitelma" -> "Luo uusi ratanumero" -> "Käyttöoikeudet eivät riitä toiminnon suorittamiseen". Lokaali käyttäjä oli operaattoriroolinen.

Reply:
Täytynee jutella tästä sitten dailyssä. Veikkaan että näitä uusia oikeuksia ei ole mietitty tätä koko ominaisuutta silmälläpitäen (enkä itse edes muistanut että täällä voi luoda ratanumeroita vaikka inframallien tuontia testasinkin.) En tosin kyllä keksi miksi operaattoriroolilla tuo luominen failaisi, mutta tutkin asiaa 🤔